addons/sale_timesheet/report/project_profitability_report_analysis.py

```diff
@@ -200,23 +217,31 @@ def init(self):
                                 FROM project_project P
                                     JOIN account_analytic_account AA ON P.analytic_account_id = AA.id
                                     JOIN account_analytic_line AAL ON AAL.account_id = AA.id
-                                    LEFT JOIN account_move_line RINVL ON AAL.move_id = RINVL.id
-                                                                     AND RINVL.parent_state = 'posted'
-                                                                     AND RINVL.exclude_from_invoice_tab = 'f'
-                                    -- Check if the AAL is not related to a reversed credit note
-                                    LEFT JOIN account_move RINV ON RINV.id = RINVL.move_id
-                                    LEFT JOIN account_move_line INVL ON INVL.move_id = RINV.reversed_entry_id
-                                                                    AND INVL.parent_state = 'posted'
-                                                                    AND INVL.exclude_from_invoice_tab = 'f'
-                                                                    AND INVL.product_id = RINVL.product_id
-                                    LEFT JOIN sale_order_line_invoice_rel SOINV ON SOINV.invoice_line_id = INVL.id
+                                    LEFT JOIN account_move_line AML ON AAL.move_id = AML.id
+                                                                     AND AML.parent_state = 'posted'
+                                                                     AND AML.exclude_from_invoice_tab = 'f'
+                                    -- Check if it's not a Credit Note for an Invoice
+                                    LEFT JOIN account_move INV ON INV.id = AML.move_id
+                                    LEFT JOIN account_move_line RINVL ON RINVL.move_id = INV.reversed_entry_id
+                                                                    AND RINVL.parent_state = 'posted'
+                                                                    AND RINVL.exclude_from_invoice_tab = 'f'
+                                                                    AND RINVL.product_id = AML.product_id
```

Comment:
I think we could keep the `RINV` alias for this account_move (which is the credit note, which is prefixed by RINV in the interface) and `INVL` for this account move line.

What do you think about it ?